### PR TITLE
Polarization

### PR DESCRIPTION
**Pull request by grburgess**
starting to add polarization capabilities.

### CODE REVIEW

File: astromodels/core/polarization.py [hunk 1]
```diff
@@ -1,10 +1,102 @@
...
 from astromodels.core.tree import Node
+from astromodels.core.parameter import Parameter
 
 
 class Polarization(Node):
 
-    def __init__(self):
+    def __init__(self, type='linear'):
+
+        assert type in ['linear', 'stokes'], 'polarization must be linear or stokes'
```
Comment (giacomov) [inline astromodels/core/polarization.py]: Please do not use `type` as variable, as it is a python protected keyword.

Comment (grburgess) [reply astromodels/core/polarization.py]: Doh! Speed coding.

**Comment by grburgess** [comment]
I fixed a bug and added imports. I'm not sure how you are doing the docs, so I just added an example in the Model tutorial docs that showed how to add polarization.

**Comment by grburgess** [comment]
I'm curious if we should allow the creation of a polarization source directly in the point source constructor or keep it as a part of the spectral component. This latter is more explicit. If you could somehow have one polarization for each spectrum and measure them separately, that would be very cool. Thoughts @giacomov ?

**Pull request merged**